politeiawww/backend.go

```diff
@@ -1317,11 +1366,41 @@ func (b *backend) ProcessLogin(l www.Login) (*www.LoginReply, error) {
 	err = bcrypt.CompareHashAndPassword(user.HashedPassword,
 		[]byte(l.Password))
 	if err != nil {
+		user.FailedLoginAttempts += 1
+		if user.FailedLoginAttempts == LoginAttemptsToLockUser {
```

Comment:
It's possible that the server is rebuilt with a smaller `LoginAttemptsToLockUser` value than one of the existing user's `FailedLoginAttempts` values, so it's probably safer to use `>=` here.